docs/sphinx/source/whatsnew/v0.7.0.rst

```diff
@@ -10,6 +10,40 @@ compatibility notes.
 **Python 2.7 support ended on June 1, 2019.** (:issue:`501`)
 **Minimum numpy version is now 1.10.4. Minimum pandas version is now 0.18.1.**
 
+API Changes
+~~~~~~~~~~~
+* Changes to functions and methods for cell temperature models:
+  (:issue:`678')
+    - Moved functions for cell temperature and constant 
+      `TEMP_MODEL_PARAMS` from `pvsystem.py` to `temperature.py`.
+    - Renamed `pvsystem.sapm_celltemp` and `pvsystem.pvsyst_celltemp`
+      to `temperature.sapm_cell` and `temperature.pvsyst_cell`.
+    - Created `temperature.sapm_module`.
```

Comment:
`temperature.sapm_cell` now returns only the cell temperature instead of a `DataFrame` of cell and module temperature.